docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -21,8 +21,8 @@ import { mergeThemeVariables } from '../../../../../src/lib/mergeThemes'
 import { ThemeContext } from '../../../context/theme-context'
 
 export interface ComponentExampleProps extends RouteComponentProps<any, any> {
-  title: string
-  description: string
+  title: React.ReactNode
+  description?: React.ReactNode
```

Comment:
`title` and `description` are simply passed to `ComponentExampleTitle` that accepts `ReactNode`, too.

Reply:
not sure about that. We should express correct semantics to the client of this component, and if we would search all the places where `<ComponentExample />` is used, all of them suggest that `string` values should be provided for these two

Reply:
Why component's description should be a `string`? I think that we should be able to pass an additional formatting there as it was in SUIR:

![image](https://user-images.githubusercontent.com/14183168/48270247-4320d600-e442-11e8-8ab7-1895c27d083c.png)

Reply:
got it - so, essentially proposal is to augment current capabilities, where `string` was previously used everywhere?

Reply:
In fact, yes. We have the same thing in SUIR docs, so it will work nicely there.

I think that the `string` type was assigned by mistake and this change just corrects the behaviour to correct.